src/pdm_multirun/plugin.py

```diff
@@ -71,9 +72,10 @@ def handle(self, project: Project, options: argparse.Namespace) -> None:  # noqa
                 if exit.code:
                     self._use(project, options, old_python)
                     raise
+            os.environ.pop("PDM_MULTIRUN_CURRENT")
+
         project.core.ui.echo(f"Restoring interpreter: {old_python}", verbosity=termui.Verbosity.DETAIL)
         self._use(project, options, old_python)
-        os.environ.pop("PDM_MULTIRUN", None)
```

Comment:
Why not?

Reply:
ah sorry - must've been deleted in error